jdl/jhipster/default-application-options.ts

```diff
@@ -185,11 +182,8 @@ export function getConfigForMicroserviceApplication(customOptions: any = {}): an
   if (typeof options[SKIP_USER_MANAGEMENT] !== 'boolean') {
     options[SKIP_USER_MANAGEMENT] = true;
   }
-  if (options[SERVICE_DISCOVERY_TYPE] === false) {
-    options[SERVICE_DISCOVERY_TYPE] = EUREKA;
-  }
   if (options[SERVICE_DISCOVERY_TYPE] === NO_SERVICE_DISCOVERY) {
-    options[SERVICE_DISCOVERY_TYPE] = false;
+    options[SERVICE_DISCOVERY_TYPE] = EUREKA;
```

Comment:
Not sure this is OK here.
I believe previous was to fix default option when not available in jdl file (undefined = false), but if it is defined as NO, we should not initialize it as Eureka.

Reply:
Fixed sir, it will modify the original behavior (setting to NO if nothing is specified), but it makes more sense doesn't it?